homeassistant/components/lgthinq/config_flow.py

```diff
@@ -0,0 +1,102 @@
+"""Config flow for LG ThinQ."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+import uuid
+
+from thinqconnect import ThinQApi, ThinQAPIException
+from thinqconnect.country import Country
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_COUNTRY
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.selector import CountrySelector, CountrySelectorConfig
+
+from .const import (
+    CLIENT_PREFIX,
+    CONF_CONNECT_CLIENT_ID,
+    DEFAULT_COUNTRY,
+    DOMAIN,
+    THINQ_DEFAULT_NAME,
+    THINQ_PAT_URL,
+)
+
+SUPPORTED_COUNTRIES = [country.value for country in Country]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ThinQFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow."""
+
+    VERSION = 1
+
+    def _get_default_country_code(self) -> str:
+        """Get the default country code based on config."""
+        return next(
+            (x for x in SUPPORTED_COUNTRIES if x == self.hass.config.country),
+            DEFAULT_COUNTRY,
+        )
```

Comment:
Can't we do a `if self.hass.config.country not in SUPPORTED_COUNTRIES: return self.hass.config.country else: return DEFAULT_COUNTRY`?

Reply:
Modified